scanpy/plotting/tools/scatterplots.py

```diff
@@ -179,10 +179,8 @@ def plot_scatter(adata,
 
     sanitize_anndata(adata)
     if color_map is not None:
-        print("instead of color_map use cmap")
```

Comment:
We would usually do this using `logg.warn` but the convention of having `size` as a parameter is also present in several of matplotlib's and seaborn's functions. I regard a completely obscure parameter name such as `s` not as good coding style. These days, we want to be verbose. So, let's not discourage people from using clear parameter names.

Reply:
I am glad that you revised this as I wanted to ask you about those options but I forgot. I agree that `s` is quite misleading. Usually, matplotlib functions have a short and a long name, but in this case only the short is present. Let's remove the warning and use `size` as in seaborn,  this needs to be added to the documentation as well. For `color_map` I don't know. Whatever is more consistent with other scanpy uses should be best.